code/modules/admin/secrets.dm

```diff
@@ -546,6 +547,23 @@
 				B.update_hair()
 			message_admins("[key_name_admin(usr)] activated dorf mode")
 
+		if("schoolgirl")
+			if(!check_rights(R_FUN))
+				return
+			feedback_inc("admin_secrets_fun_used",1)
+			feedback_add_details("admin_secrets_fun_used","SG")
+			for(var/obj/item/clothing/under/W in world)
```

Comment:
>looping through everything in world

totally isn't going to lag

be smart
add a global list of just under clothing for looping through